python/samples/concepts/plugins/crew_ai_plugin.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+import logging
+
+from semantic_kernel.core_plugins.crew_ai.crew_ai_enterprise import CrewAIEnterprise, InputMetadata
+from semantic_kernel.core_plugins.crew_ai.crew_ai_settings import CrewAISettings
+from semantic_kernel.functions.kernel_arguments import KernelArguments
+from semantic_kernel.functions.kernel_function import KernelFunction
+from semantic_kernel.kernel import Kernel
+
+logging.basicConfig(level=logging.INFO)
+
+
+async def using_crew_ai_enterprise():
```

Comment:
I would combine these two into a single sample, that first calls the crew directly and then creates the plugin and calls that